koji_containerbuild/cli.py

```diff
@@ -103,7 +102,23 @@ def handle_container_build(options, session, args):
         parser.error(_("Exactly two arguments (a build target and a SCM URL "
                        "or archive file) are required"))
         assert False
-    clikoji.activate_session(session)
+
+    # Koji API has changed - activate_session requires two arguments
+    # _running_in_bg has been moved to koji_cli.lib
+    try:
+        from koji_cli.lib import _running_in_bg, activate_session
+    except ImportError:
+        # Create wrappers for backwards compatibility.
+        _running_in_bg = clikoij._running_in_bg
```

Comment:
Created https://github.com/release-engineering/koji-containerbuild/pull/72 to fix that